docs/document/static/js/samples/desktop/user-org-group-select.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef } from "react";
+
+export const UserOrgGroupSelectComponent = () => {
+  const divEl = useRef();
+  useEffect(() => {
+    const userOrgGroupSelect = new Kuc.UserOrgGroupSelect({
+      label: "Sample label",
+      requiredIcon: true,
+      items: [
```

Comment:
Please use the reasonable user, org, group name.
e.g. Sales department, John Brown, etc